rest_configuration.cpp

```diff
@@ -1079,19 +1075,30 @@ void DeRestPluginPrivate::configToMap(const ApiRequest &req, QVariantMap &map)
 
 /*! Puts all parameters in a map for later JSON serialization.
  */
-void DeRestPluginPrivate::basicConfigToMap(QVariantMap &map)
+void DeRestPluginPrivate::basicConfigToMap(const ApiRequest &req, QVariantMap &map)
 {
     map["name"] = gwName;
-    map["datastoreversion"] = QLatin1String("93");
-    const QStringList versions = QString(GW_SW_VERSION).split('.');
-    const QString swversion = QString("%1.%2.%3").arg(versions[0].toInt()).arg(versions[1].toInt()).arg(versions[2].toInt());
-    map["swversion"] = swversion;
-    map["apiversion"] = QString(GW_API_VERSION);
+    map["modelid"] = QLatin1String("deCONZ");
+    if (req.mode == ApiModeNormal)
+    {
+        const QStringList versions = QString(GW_SW_VERSION).split('.');
+        const QString swversion = QString("%1.%2.%3").arg(versions[0].toInt()).arg(versions[1].toInt()).arg(versions[2].toInt());
+        map["swversion"] = swversion;
+        map["apiversion"] = QString(GW_API_VERSION);
+        // map["modelid"] = QLatin1String("deCONZ");
+        map["datastoreversion"] = QLatin1String("93");
+    } 
+    else
+    {
+        map["swversion"] = QLatin1String("1941132070");
+        map["apiversion"] = QLatin1String("1.41.0");
+        // map["modelid"] = QLatin1String("BSB002");

```

Comment:
Since the Hue "modelid" was remove above it should to be set here I guess?

Reply:
"modelid" is set in line 1081 to "deCONZ", it seems that the Hue App at least doesn't care that it's not "BSB002".
I at least can remove the commented lines though...

Reply:
Good to know, the Hue App might be fine but have you tested this also with Amazon Echo?
If in doubt I'd vote to just keep the BSB002 `modelid`.

Reply:
I haven't tested it directly with the Echo, I'm using Alexa integration through homebridge. Just to be sure I've put back the BSB002 modelid.